apps/src/javalab/JavalabView.jsx

```diff
@@ -87,7 +87,12 @@ class JavalabView extends React.Component {
             className="responsive"
             style={styles.instructionsAndPreview}
           >
-            <TopInstructions mainStyle={styles.instructions} standalone />
+            <TopInstructions
```

Comment:
How does overflow behave? i.e. the tabs are starting to get verbose. If the visualization area is shrunk down, do the tabs behave well?

Reply:
looks like we will need to update the width of the instructions (make them wider) are based on the design spec, so this may be less of a concern?

Reply:
molly's right -- the current default width of the instructions/preview portions needs to be increased to match the design. horizontal overflow does cause a scrollbar to show up so everything is still accessible, but the header isn't tall enough for that to actually be a good experience:

<img width="349" alt="Screen Shot 2021-05-26 at 10 10 14 AM" src="https://user-images.githubusercontent.com/9812299/119702719-9e512300-be0a-11eb-8e8c-cded3ba0a307.png">

Reply:
Thinking about Maze, it has a rather small maximum size - and we'll still want to support the horizontal resizer. Also, thinking about internationalization, we may want some way to handle longer words. i.e. those three in Norwegian: Bruksanvisning dokumentasjon anmeldelse. 
This is a conversation that probably needs to be resolved by Mike & Mark outside this PR. Definitely non-blocking.

Reply:
^ There's also a tab "For Teachers Only" that shows up if you're logged in with a teacher account.